apps/studio/src/templates/layouts/AdminCmsSidebarLayout.tsx

```diff
@@ -0,0 +1,72 @@
+import { useRouter } from "next/router"
+import { Flex } from "@chakra-ui/react"
+import {
+  BiCog,
+  BiFolder,
+  BiGroup,
+  BiHelpCircle,
+  BiLinkExternal,
+  BiLogOut,
+} from "react-icons/bi"
+
+import type { CmsSidebarItem } from "~/components/CmsSidebar/CmsSidebarItems"
+import { EnforceLoginStatePageWrapper } from "~/components/AuthWrappers"
+import { CmsSidebar, CmsSidebarContainer } from "~/components/CmsSidebar"
+import { useMe } from "~/features/me/api"
+import { type GetLayout } from "~/lib/types"
+
+export const AdminCmsSidebarLayout: GetLayout = (page) => {
+  const router = useRouter()
+  const siteId = String(router.query.siteId)
+
+  const { logout } = useMe()
+
+  const pageNavItems: CmsSidebarItem[] = [
+    {
+      icon: BiFolder,
+      label: "Team spaces",
+      href: `/sites/${siteId}`,
+      isActive:
+        router.asPath === `/sites/${siteId}` ||
+        router.asPath.startsWith(`/sites/${siteId}/pages`),
+    },
+    { icon: BiGroup, label: "Manage users", href: `/sites/${siteId}/manage` },
+    { icon: BiCog, label: "Settings", href: `/sites/${siteId}/settings` },
+  ]
+
+  const userNavItems: CmsSidebarItem[] = [
+    {
+      icon: BiLinkExternal,
+      label: "Open live site",
+      // TOOD: Replace with actual live site URL
+      href: `/home`,
+    },
+    {
+      icon: BiHelpCircle,
+      label: "Isomer Guide ",
+      href: "https://guide.isomer.gov.sg/",
+    },
+    {
+      icon: BiLogOut,
+      label: "Sign out",
+      onClick: logout,
+    },
+  ]
+
+  return (
+    <EnforceLoginStatePageWrapper>
+      <Flex minH="$100vh" flexDir="row" bg="base.canvas.alt" pos="relative">
+        <CmsSidebarContainer
+          sidebar={
+            <CmsSidebar
+              topNavItems={pageNavItems}
+              bottomNavItems={userNavItems}
+            />
```

Comment:
tbh, not a fan of this pattern over using something like a `children` prop. not too big a deal because i feel this section of the codebase is unlikely to change